codesign/lib/codesign.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
```

Comment:
I believe you should put 2019 as the copyright year, as that is the copyright year of the repo (I am not a lawyer, but this is what Hixie tells us to do in flutter/flutter :) )